diff --git a/llm/generate/gen_common.sh b/llm/generate/gen_common.sh
index 3825c155..d22eccd2 100644
--- a/llm/generate/gen_common.sh
+++ b/llm/generate/gen_common.sh
@@ -69,6 +69,8 @@ git_module_setup() {
 }
 
 apply_patches() {
+    return
+
     # apply temporary patches until fix is upstream
     for patch in ../patches/*.patch; do
         git -c 'user.name=nobody' -c 'user.email=<>' -C ${LLAMACPP_DIR} am ${patch}
@@ -133,6 +133,8 @@ install() {
 
 # Keep the local tree clean after we're done with the build
 cleanup() {
+    return
+
     git submodule update --force ${LLAMACPP_DIR}
 }
